dataland-frontend/src/components/forms/CreateCompany.vue

```diff
@@ -1,130 +1,382 @@
 <template>
```

Comment:
Was there a design in the figma for this page?

Reply:
Not for the page itself, but for the company upload form.

Check out the following figma and scroll to the second window from the left:

https://www.figma.com/file/0tgJDwGCXQxxnLvFRWgaRu/Mockup-v06.3-(09%2F09%2F2022)---Data-Collection?node-id=2480%3A4298&t=NpEtZvclXrGMcJ1A-0

Reply:
Thank you, then i use this to compare

Reply:
Ok, i resolve it